app/v1/end_user.pb.go

```diff
@@ -182,6 +182,123 @@ func (*AcceptLegalResponse) Descriptor() ([]byte, []int) {
 	return file_app_v1_end_user_proto_rawDescGZIP(), []int{3}
 }
 
+type RegisterApplicationRequest struct {
+	state         protoimpl.MessageState
+	sizeCache     protoimpl.SizeCache
+	unknownFields protoimpl.UnknownFields
+
+	ApplicationName string   `protobuf:"bytes,1,opt,name=application_name,json=applicationName,proto3" json:"application_name,omitempty"`
+	ApplicationId   string   `protobuf:"bytes,2,opt,name=application_id,json=applicationId,proto3" json:"application_id,omitempty"`
+	OrgId           string   `protobuf:"bytes,3,opt,name=org_id,json=orgId,proto3" json:"org_id,omitempty"`
+	OriginUris      []string `protobuf:"bytes,4,rep,name=origin_uris,json=originUris,proto3" json:"origin_uris,omitempty"`
+	RedirectUris    []string `protobuf:"bytes,5,rep,name=redirect_uris,json=redirectUris,proto3" json:"redirect_uris,omitempty"`
+}
```

Comment:
@ohEmily could you weigh in on these params and if they make sense?

Reply:
Curious about `omitempty`. I thought we shouldn't do `omitempty` for `OriginUris`, `RedirectUris`, `OrgId` and `ApplicationName`, unless we intend to do app-side validation for those. Wouldn't `omitempty` mean that the fields are optional? Out of these, I believe `ApplicationId` is the only optional one that should have `omitempty`.